requests-for-comments/2023-05-30-clickhouse-as-only-store.md

```diff
@@ -0,0 +1,35 @@
+# Request for comments: Clickhouse as our only data store
+
+## Problem statement
+
+- Having two separate places where we store essentially the same data (person/person_distinct_id/groups) means we often have inconsistencies
```

Comment:
Postgres is the source of truth, and this inconsistency is that clickhouse data is sometimes wrong - we'll need to figure out a way to fix that for this to work.